Examples/Example-macOS-ObjC/AppDelegate.m

```diff
@@ -41,6 +43,11 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
     } @finally {
         
     }
+    NSDictionary *attributes = @{@"My Attribute":@"My Attribute Value"};
+    [BacktraceClient.shared addBreadcrumb:@"My Native Breadcrumb"
```

Comment:
does it make sense to add that this is a native breadcrumb? All breadcrumbs are from the native layer here.

Reply:
It's just the example app, but we can adjust it

Reply:
Done.